GTG/gtk/browser/browser.py

```diff
@@ -502,53 +524,6 @@ def open_task(req, t):
         for t in self.config.get("opened_tasks"):
             GObject.idle_add(open_task, self.req, t)
 
-    def do_toggle_workview(self):
-        """ Switch between default and work view
-
-        Updating tags is disabled while changing view. It consumes
-        a lot of CPU cycles and the user does not see it. Afterwards,
-        updating of tags is re-enabled and all tags are refreshed.
-
-        Because workview can be switched from more than one button
-        (currently toggle button and check menu item), we need to change
-        status of others also. It invokes again this method =>
-        a loop of signals.
-
-        It is more flexible to have a dedicated variable
-        (self.in_toggle_workview) which prevents that recursion. The other way
-        how to solve this is to checking state of those two buttons and check
-        if they are not the same. Adding another way may complicate things...
-        """
-
-        if self.in_toggle_workview:
-            return
-
-        self.in_toggle_workview = True
-
-        if self.config.get('view') == 'workview':
-            self.set_view('default')
-        else:
-            self.set_view('workview')
-
-        self.in_toggle_workview = False
-
-    def refresh_workview(self, timer):
-        task_tree = self.req.get_tasks_tree(name='active', refresh=False)
-        task_tree.refresh_all()
-
-    def set_view(self, viewname):
-        if viewname == 'default':
-            self.req.unapply_global_filter(self.activetree, 'workview')
```

Comment:
Can you put an comment next to GTG/core/requester.py for unapply_global_filter & apply_global_filter to evaluate if they are used anywhere?

Spoiler alert: they are not, but might be in later.